src/prognostic_equations/edmfx_sgs_flux.jl

```diff
@@ -118,7 +176,19 @@ function edmfx_sgs_flux_tendency!(
             )
         end
 
+        # diffusive flux
+        ᶠρaK_h = p.ᶠtemp_scalar
+        @. ᶠρaK_h[colidx] = ᶠinterp(Y.c.ρ[colidx]) * ᶠinterp(ᶜK_h[colidx])
+
+        ᶜdivᵥ_ρe_tot = Operators.DivergenceF2C(
```

Comment:
Actually maybe the allocation is from this. Is this ok @charleskawczynski ?

Reply:
Maybe? Can you check to see if the number of jet failures is changed or unchanged? (It’s printed in a dictionary in the log)

Reply:
Ah, it is changed. It changes from 
```
"limited_tendency!" => 8
"remaining_tendency!" => 8
"set_precomputed_quantities!" => 8
"step!" => 235
"implicit_tendency!" => 8
```
to
```
"additional_tendency!" => 2
"limited_tendency!" => 8
"remaining_tendency!" => 10
"set_precomputed_quantities!" => 8
"step!" => 236
"implicit_tendency!" => 8
```
Note the command line for the target job also changed (it changed from --vert_diff true to false(default), see the pipeline change).

Reply:
I guess it means there are type instabilities in my code?

Reply:
Yes, we can modify https://github.com/CliMA/ClimaAtmos.jl/blob/main/perf/jet.jl#L5 to point to the breaking job in order to identify where we're adding type-unstable code.